eq-author/src/components/buttons/ToggleSwitch/__snapshots__/index.test.js.snap

```diff
@@ -365,7 +365,7 @@ exports[`ToggleSwitch should render when checked and mounted 1`] = `
 .c4 {
   height: 1em;
   width: 1.5em;
-  top: 1em;
+  top: 0.6000000000000001em;
```

Comment:
WAT?

Reply:
Some styles have changed in the `ToggleSwitch` as per @hamishtaplin prototype.  This is probably calculated or something.

Reply:
What is the calculation? Is it a css `calc` or something done in JS that we could move into the code? Or maybe I've miss understood this

Reply:
https://github.com/ONSdigital/eq-author-app/blob/master/eq-author/src/components/buttons/ToggleSwitch/index.js#L34

Reply:
Ah, no its me being stupid. This is a snaphshot! I'm an idiot. The calculation is there.